Add new boolean option "use_fstab".

BrHal opened the pull request:
Hi, After more complete code reading, I found that using `/etc/fstab` could be helpful.
I have added the `use_fstab` option defaulting to `False` as well as the disk list filtering. It is compatible with existing filters. The list is auto refreshed by mounting/unmounting (was already) as well as `/etc/fstab` modifications.
For now, using it with non-existing `/etc/fstab` filters out all disks.

aristocratos commented on bpytop.py:
```diff
@@ -3069,10 +3074,22 @@ def _collect(cls):
 					errlog.error(f'Caused by outdated psutil version.')
 				errlog.exception(f'{e}')
 			io_counters = None
+		try:
```
Should probably add some conditions to this block since it doesn't need to run if `CONFIG.use_fstab == False` and also if `SYSTEM == "MacOS"` since OsX don't use `/etc/fstab`.

For performance you could possibly store `fstab_filter` as a class variable also and only run the code block if the list is empty. `/etc/fstab` should be unlikely to change right?

BrHal replied:
Hi, thanks for the review, I will be updating for optimization.
My first idea was to hide the option if `/etc/fstab` does not exist, sticking it to `False`, however if I find no clean way to do this, I'll keep on your idea.
Agreed about the refresh based on `/etc/fstab` changes, it was rather a test case for me.

aristocratos commented:
@BrHal 
Great idea! And nice work.

aristocratos commented:
@BrHal 
Looks good, will merge in next update.
Thanks for contributing!

BrHal commented:
Hi, 
fstab_filter now moved to cls level, updated only once as flag is turned on, cleared if flag is turned off.
Flag automatic reset to false in case of `/etc/fstab` read failure. 
Plus debug logging.
